### PR TITLE
Type default value check

### PR DESCRIPTION
**Pull request by paulr34**

### CODE REVIEW

File: src-electron/util/types.js [hunk 1]
```diff
@@ -276,20 +278,13 @@ function isFloat(type) {
...
-    case 'int48s':
-    case 'int56s':
-    case 'int64s':
-      return true
-    default:
-      return false
-  }
+async function isSignedInteger(db, sessionId, type) {
+  let sessionPackages = await queryPackages.getSessionPackages(db, sessionId)
+  return await queryAtomic.isAtomicSignedByNameAndPackage(
```
Comment (brdandu) [inline src-electron/util/types.js:283]: You do not need this new function we have code to retrieve this using query-zcl.js#selectNumberByName

**Review by brdandu — CHANGES_REQUESTED** [review]

File: src-electron/util/types.js (hunk 1 above)
Comment (paulr34) [reply src-electron/util/types.js:283]: oh thanks

File: src-electron/generator/helper-endpointconfig.js [hunk 2]
```diff
@@ -645,22 +645,23 @@ function asMEI(manufacturerCode, code) {
...
     return specifiedDefault
   }
 
   if (types.isString(type)) {
     // Handled elsewhere.
     return null
   }
-
-  if (types.isSignedInteger(type)) {
+  if (await types.isSignedInteger(db, sessionId, type)) {
```
Comment (brdandu) [inline src-electron/generator/helper-endpointconfig.js:664]: This signature has not been changed in `validation.js -> types.isSignedInteger(attribute.type)`

Comment (paulr34) [reply src-electron/generator/helper-endpointconfig.js:664]: ohh I see
thank you

**Pull request closed without merging**